controllers/apply/under10k/fields/organisation-finances.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+const get = require('lodash/fp/get');
+const { oneLine } = require('common-tags');
+
+const Joi = require('../../lib/joi-extensions');
+const { CurrencyField } = require('../../lib/field-types');
+const isNewOrganisation = require('../lib/new-organisation');
+
+module.exports = {
```

Comment:
Preferring this pattern of grouping fields by section rather than one file per field which is a little overkill.